grommet/grommet
# Update NameValueList to fix propType errors

#### What does this PR do?
Makes adjustments to NameValueList to fix proptype errors at small viewport. Additionally, adds default behavior of `[auto, valueWidth]` to fix cases where value would exceed its container.

#### Where should the reviewer start?
src/js/components/NameValueList/NameValueList.js

#### What testing has been done on this PR?
Storybook local.

#### How should this be manually tested?
Launch storybook in dev mode and adjust viewport to small width and confirm there are no console errors.

#### Do Jest tests follow these best practices?

N/A, didn't touch tests.


#### Any background context you want to provide?

#### What are the relevant issues?
Discovered in https://github.com/grommet/hpe-design-system/pull/2164

#### Screenshots (if appropriate)

#### Do the grommet docs need to be updated?
No.

#### Should this PR be mentioned in the release notes?
Yes.

#### Is this change backwards compatible or is it a breaking change?
Backwards compatible. Fixes bug where value was overflowing container.
